.gitattributes

```diff
@@ -1,5 +1,5 @@
 # Set default behaviour, in case users don't have core.autocrlf set.
-* text=auto
+* text=input
```

Comment:
We want to always convert to LF on commit to Git but leave it up to eol settings declared below and the native system to handle checkout. This should prevent this issue from coming up again in the future, as we won't be able to have files committed with CRLF endings as seen in the other file change in this PR.